readthedocs/projects/tasks.py

```diff
@@ -239,17 +230,10 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
         return False
 
 
-# TODO UpdateDocsTask should be refactored into a standard celery task,
-# there is no more need to have this be a separate class
-class UpdateDocsTask(Task):
-
-    max_retries = 5
-    default_retry_delay = (7 * 60)
-    name = __name__ + '.update_docs'
-
-    def run(self, *args, **kwargs):
-        step = UpdateDocsTaskStep(task=self)
-        return step.run(*args, **kwargs)
+@app.task(bind=True, max_retries=5, default_retry_delay=7 * 60)
+def update_docs_task(self, project_id, *args, **kwargs):
```

Comment:
Pylint doesn't like binded task, the self parameter gets injected by the decorator.